prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100% / 3);
```

Comment:
Since `flex-basis` only affects "flex items", we'll want to move this declaration to a rule set for `.card-wrapper` elements.